src/components/book.jsx

```diff
@@ -4,19 +4,27 @@ import Book from './home';
 import Form from './form';
 import { getBooks } from '../redux/book/book';
 
+import './book.css';
+
 const Books = () => {
   const dispatch = useDispatch();
   const books = useSelector((state) => state.books);
   useEffect(() => {
     dispatch(getBooks());
   }, [dispatch]);
   return (
-    <>
+    <div className="book">
       {books.map((book) => (
-        <Book key={book.item_id} id={book.item_id} title={book.title} author={book.author} />
+        <Book
+          key={book.item_id}
+          id={book.item_id}
+          title={book.title}
+          author={book.author}
+          category={book.category}
+        />
       ))}
       <Form />
```

Comment:
- [ ] Let's once more keenly take an observation at the differences between our design and that of the original below:

| Yours | Original |
| --- | --- |
| ![annotely_image (11)](https://user-images.githubusercontent.com/63087491/202350950-3d8ba6b8-295f-46c6-8bb8-209dc1a132c0.png) | ![zeplin](https://user-images.githubusercontent.com/63087491/202351011-165ef6cc-3fa7-4bc8-9ef3-d85fc57f0629.png) |

- Based on the requirements of this project, your stylings should be EXACTLY the same as that in Zepline(fonts, margins, colors, orientations, and layouts should be exactly the same as that in Zeplin)
- I have done some annotations on your design. You'll kindly compare it with the original design and make the necessary adjustments. Seems you forgot to include the user icon. Your logo(Bookstore), Books, and Categories on the navbar should have the same font as in the original design. Please once more check out the colors and margins because I found some differences as you can see in the annotations above.
-